.github/workflows/continuous-integration-workflow.yaml

```diff
@@ -103,15 +103,14 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v3
-        with:
-          submodules: recursive
       - name: Verify with Kani
-        uses: model-checking/kani-github-action@v0.19
+        uses: model-checking/kani-github-action@0.23
         with:
           enable-propproof: true
           args: |
             --tests -p prost-types --default-unwind 3 \
-            --harness "tests::check_timestamp_roundtrip_via_system_time"
+            --harness "tests::check_timestamp_roundtrip_via_system_time" \
+            --harness "tests::check_duration_roundtrip_nanos"
```

Comment:
It would be nice if we could include kani tests via some sort of wildcard instead of listing each test.

Reply:
Actually, this can already be done since we upgraded to `0.23`. The `--harness <harness>` argument will run any harnesses for which `<harness>` is a sub-string will be run (i.e., it works like `cargo test --test <test>`). 

Unfortunately, `--harness tests::check` matches other tests which take much longer to verify. Because of that, I'd suggest leaving full names for now.